### PR TITLE
Polish language

### PR DESCRIPTION
**Pull request by AndrzejRPiotrowski**
Adding new language to use as work.
Need to use this as PoC for non English/French Speakres

### CODE REVIEW

**Comment by AndrzejRPiotrowski** [comment]
I have read the CLA Document and I hereby sign the CLA

File: frontend/src/lib/components/SideBar/SideBarFooter.svelte [hunk 1]
```diff
@@ -17,6 +17,7 @@
 		german: m.german(),
 		dutch: m.dutch(),
 		italian: m.italian()
```
Comment (Mohamed-Hacene) [inline frontend/src/lib/components/SideBar/SideBarFooter.svelte]: the missing comma may be the cause of failed tests

File: frontend/messages/fr.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
 	"german": "Allemand",
 	"dutch": "Néerlandais",
 	"italian": "Italien",
+	"polish" :"Polonge",
```
Comment (Mohamed-Hacene) [inline frontend/messages/fr.json]: "Polonais"

**Review by Mohamed-Hacene — CHANGES_REQUESTED** [review]

File: frontend/src/lib/components/SideBar/SideBarFooter.svelte (hunk 1 above)
Comment (AndrzejRPiotrowski) [reply frontend/src/lib/components/SideBar/SideBarFooter.svelte]: Ok great

File: frontend/messages/fr.json (hunk 2 above)
Comment (AndrzejRPiotrowski) [reply frontend/messages/fr.json]: Thx

**Comment by AndrzejRPiotrowski** [comment]
Changes made

**Comment by ab-smith** [comment]
We can rerun the formatter afterwards, thanks @AndrzejRPiotrowski for the contribution 😊

**Pull request merged**